data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg

```diff
@@ -170,7 +170,7 @@
 
         [message]
             speaker=Tallin
-            message= _ "With all respect, my Lord Hamel, we the people from Dwarven Doors have faced certain death many times now. Rising up against the orcs was said to be certain death. Plunging headfirst into these bleak caves was said to be certain death. If our fate is death then so be it. Until that time comes, my people will fight and I will lead them to fight!"
+            message= _ "With all respect, my Lord Hamel, we the people from Dwarven Doors have faced certain death many times now. Rising up against the orcs was said to be certain death. Plunging headfirst into these bleak caves was said to be certain death. If our fate is death, then so be it. Until that time comes, my people will fight, and I will lead them."
```

Comment:
The first added comma seems okay, but to me it doesn't need a comma there - it would have a slight pause there when spoken, yet it's a significant enough statement that the stress added by the reader is more important than the comma, and so it doesn't need the comma.

For the second sentence I prefer the original.

Reply:
I honestly don't think the "to fight!" adds anything, nor does it sound natural. If someone else agrees with you though, I'll change it.